### PR TITLE
Improve doc about js demo

### PR DESCRIPTION
**Pull request by hzz780**

### CODE REVIEW

File: docs/tutorials/dapp/setup.md [hunk 1]
```diff
@@ -112,6 +112,18 @@ npm start
 
 You should see the results in the terminal or in the browser dev tool.
 
+### In the terminal
+
+<p align="center">
+  <img src="assets/jssdk_terminal.jpg" width="400">
+</p>
```
Comment (rosona) [inline docs/tutorials/dapp/setup.md]: try this format?  avatar

Comment (loning) [reply docs/tutorials/dapp/setup.md]: png or jpg is ok

Comment (loning) [reply docs/tutorials/dapp/setup.md]: maybe you just need to copy the output, and put into 
```
OUTPUT
```

Comment (hzz780) [reply docs/tutorials/dapp/setup.md]: eeee, a good idea.
But image may be more intuitive.

**Pull request merged**